DESIGN.md

```diff
@@ -74,4 +74,35 @@ It is "hot" because consumers subscribing to it does not cause side-effects, or
 
 A type representing work that can be cancelled or disposed.
 
+### Behavior
+
+##### Creation
+
+Creation of a stream falls into the following use cases, all of which should be catered to in API design.
+
+- async, hot, push (ie. system or user events)
+- async, cold, push (ie. events resulting from remote system via network connection)
```

Comment:
I don't understand your point. Rx does not involve itself in network behaviors. What are you suggesting should be changed in the text.